src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -594,6 +604,18 @@ public String getInstructorFeedbackRemindParticularStudentsLink(String courseID,
         return link;
     }
     
+    /**
+     * Retrieves the link to submit the request for remind particular student
+     * @param returnUrl the url to return to after submitting the request
+     * @return submit link with return url appended to it
+     */
+    public String getInstructorFeedbackRemindParticularStudentActionLink(String returnUrl) {
```

Comment:
Is there a reason for adding "Action" in the name? Most of the methods don't seem to use that convention.
Also, "Students" would be more accurate. (likewise for the comment)